setup.py

```diff
@@ -1,12 +1,5 @@
 import sys
-from distutils.core import setup
-
-if sys.version_info[:2] < (2, 5) or sys.version_info[0] > 2:
-    msg = ("LabJackPython requires Python 2.5 or later but does not work on "
-           "any version of Python 3.  You are using version %s.  Please "
-           "install using a supported version." % sys.version)
-    sys.stderr.write(msg)
-    sys.exit(1)
+from setuptools import setup
```

Comment:
I prefer that additional packages do not need to be required for installation, but I don't mind using setuptools if available. For backwards compatibility is should be something like this:

try:
    from setuptools import setup
except ImportError:
    from distutils.core import setup

And keep the Python version check.